src/social-network-provider/twitter.com/ui/injectKnownIdentity.tsx

```diff
@@ -33,7 +33,8 @@ export function PersonKnownAtTwitter(props: PersonKnownProps) {
         />
     )
 }
-
+// Todo: we could remove it as it is unused since we no longer need to paste public key to bio.
+// Todo: But the code itself may be valuable, so keep it temporarily. - zhouhancheng
```

Comment:
You can always find the code in the git history. No need to leave it here.

Reply:
Okay